wapitiCore/attack/mod_log4shell.py

```diff
@@ -173,12 +209,11 @@ def _generate_payload(self, unique_id: uuid.UUID) -> str:
         # but it is also using this syntax for the exploit.
         return "${jndi:dns://" + f"{self.dns_endpoint}/{unique_id}" + ".l}"
 
-    @staticmethod
-    def _is_valid_dns(dns_endpoint: str) -> str:
+    def _is_valid_dns(self, dns_endpoint: str) -> str:
```

Comment:
Fix annotation

Reply:
and also the return type of `_get_malicious_headers` is not coherent with the one of `batch_malicious_headers`

Reply:
I removed the annotation because in the `_is_valid_dns` function, I modify the value of `_dns_host`.

Reply:
But the return type of `_is_valid_dns` should be `bool` not `str`

Reply:
Yes you're right, it's fixed